nexus/src/external_api/http_entrypoints.rs

```diff
@@ -214,7 +214,12 @@ pub fn external_api() -> NexusApiDescription {
  */
 
 /**
- * List all organizations.
+ * List all organizations
+ *
+ * Returns a list of all organizations in the system. The organizations are returned
+ * in sorted order, with the most recent organizations appearing first.
+ *
+ * TODO: figure out if the above is correct or what the order actually is.
```

Comment:
@davepacheco @ahl I could find it in the code likely but are all endpoints returned in the same order by default and what is that order